joomla/joomla-cms
# [4.0] Custom Elements everywhere: iframe wrapper

Pull Request for Issue # .

### Summary of Changes
Changes for the com_wrapper and mod_wrapper to use custom elements.

This fixes:
- Inability to use multiple modules in the same page (id was hardcoded)
- New code follows the progressive enhancement (part of the PWA) technic, eg iframe will be viewable (but not with the desired height) if javascript is disabled.
- The javascript is not blocking the rendering of the page as (and this goes for all custom elements) it's loaded asynchronously **after** the initial page render!

### Testing Instructions (by @C-Lodder)

- Create 2 wrapper modules
- assign to the same page
- ensure they work as expected


### Expected result



### Actual result
![screen shot 2017-11-29 at 16 23 27](https://user-images.githubusercontent.com/3889375/33382748-dda73196-d521-11e7-810e-437bb5c88dc7.png)



### Documentation Changes Required

None. The new code reflects the old behaviour!
